app/controllers/admin/conferences_controller.rb

```diff
@@ -61,11 +61,19 @@ def index
 
     def new
       @conference = Conference.new
+      @organizations = {}
+      Organization.all.each do |organization|
+        @organizations.store(organization.name, organization.id) if can? :create, Conference.new(organization: organization)
+      end
     end
 
     def create
-      @conference = Conference.new(conference_params)
-
+      conference_params_copy = conference_params
+      if ENV['ORGANIZATIONS_ENABLED'] != 'true'
+        org = Organization.exists?(name: 'default') ? Organization.find_by(name: 'default') : Organization.create(name: 'default')
```

Comment:
You can just use [`find_or_create_by`](http://api.rubyonrails.org/v4.2.5/classes/ActiveRecord/Relation.html#method-i-find_or_create_by).